src/taco-dependency-installer/installers/androidPackagesInstaller.ts

```diff
@@ -77,26 +82,25 @@ class AndroidPackagesInstaller extends InstallerBase {
         }
     }
 
-    private getAvailableAndroidPackages(): Q.Promise<any> {
-        var deferred: Q.Deferred<any> = Q.defer();
+    private executeAndroidCommand(name: string, args: string[]): IExecuteCommandResult {
```

Comment:
If we are just going to use this executeAndroidCommand method for these two commands, I think it'd be okay to move the y/n logic inside the command and use a boolean parameter to trigger it.

If we are going to have additional commands in the future, and they are not going to need this y/n logic, I think I'd be best to leave it out of this method.

Do you know if we have any plans to update this code, and add additional calls to the android commands?